services/processor/native/adapter/native_compiler_test.go

```diff
@@ -37,9 +37,12 @@ func TestCompileCodeWithExistingArtifacts(t *testing.T) {
 
 	t.Log("Build fresh artifacts")
 
-	sourceFilePath, err := writeSourceCodeToDisk("testPrefix", code, tmpDir)
+	sourceFilePath, err := writeSourceCodeToDisk("testPrefix", []string{code}, tmpDir)
```

Comment:
please add a test for compilation of a contract with multiple files

Reply:
It is right here:

https://github.com/orbs-network/orbs-network-go/pull/1225/files/42b4077a127425d030c224324a31cc9a3432a8a5#diff-dca888d44e4077264d0e31da69b13522R27

Unfortunately, it does not run automatically. But it is covered by e2e which compiles both single file and multi file contracts.